server/api/Dockerfile

```diff
@@ -6,14 +6,20 @@ LABEL git.branch=${BRANCH}
 LABEL git.commit=${COMMIT}
 
 ENV INSTALL_PATH /app
-RUN mkdir -p $INSTALL_PATH
-
 WORKDIR $INSTALL_PATH
 
 ENV BRANCH=${BRANCH}
 ENV COMMIT=${COMMIT}
 
-COPY requirements.txt src ./
+# TODO: Download yagna inside container
+COPY *.deb ./
```

Comment:
```
jbetley:~/yagna-service-poc/server/api {pnowosie/pass-requests-to-classificators-APPS-83} $ make build           
(...)
Step 10/15 : COPY *.deb ./
COPY failed: no source files were specified
make: *** [Makefile:8: build] Error 1
```

Please add yagna download to Makefile, the same way as in `yagna` container (or at least some message etc).